packages/base64/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@endo/base64",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Everything else start from "1.0.1", right? Why was this one "1.0.0" different?

Reply:
I’ve verified that 1.0.0 is the latest published version. Because it is a leaf dependency, none of its dependencies were altered in the last release, so it didn’t get bumped: https://github.com/endojs/endo/pull/1911

The real question is why it’s getting bumped this time since the circumstances are the same. I have no idea.